inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp

```diff
@@ -52,7 +31,52 @@ void reorder_inputs::run(program& p) { run(p, _lo, _rf); }
 namespace {
 
 std::map<program_node*, format::type> get_preferred_formats(program& p, layout_optimizer& lo) {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+
     std::map<program_node*, format::type> fmt_map;
+
+#ifdef ENABLE_ONEDNN_FOR_GPU
+    size_t onednn_impls_counter = 0;
+    size_t all_impls_counter = 0;
+    float onednn_min_threshold = 0.1f;
```

Comment:
What about `const float` instead of `float` as it is a magic number?

Reply:
Agree. Fixed.